endpoints/get.go

```diff
@@ -63,12 +63,54 @@ func writeGetResponse(w http.ResponseWriter, id string, value string) (error, in
 		w.Header().Set("Content-Type", "application/json")
 		w.Write([]byte(value)[len(backends.JSON_PREFIX):])
 	} else {
-		return fmt.Errorf("uuid=%s: Cache data was corrupted. Cannot determine type.", id), http.StatusInternalServerError
+		return fmt.Errorf("Cache data was corrupted. Cannot determine type."), http.StatusInternalServerError
 	}
 	return nil, http.StatusOK
 }
 
-func respondAndLogError(w http.ResponseWriter, err error, status int) {
-	log.Errorf(err.Error())
-	http.Error(w, err.Error(), status)
+// Will prefix error messages with "GET /cache" and, if uuid string list is passed, will
+// follow with the first element of it in the following fashion: "uuid=FIRST_ELEMENT_ON_UUID_PARAM".
+// Expects non-nil error
+func handleException(w http.ResponseWriter, err error, status int, uuid string) {
```

Comment:
If you'd like to keep the comment, please follow standard Go conventions of beginning with the method name:
```
// handleException will prefix error messages...
```